### PR TITLE
fixed parsing error; support multiple ADXL362 devices on different CE…

### PR DESCRIPTION
**Pull request by Human**
… pins

### CODE REVIEW

File: ADXL362.py [hunk 1]
```diff
@@ -156,9 +167,9 @@ def check_all_regs(self):
...
-       ''' Returns two's complement of value given a number of bits
-       '''
-        if( (val&(1<<(bits-1))) != 0 ):
-             val = val - (1<<bits)
-        return val
+       """ Returns two's complement of value given a number of bits
+       """
+       if val&(1<<(bits-1)) != 0:
+           val = val - (1<<bits)
+       return val
```
Comment (natetarrh) [inline ADXL362.py:173]: let's keep the indent at 4 spaces and fix the docstring indent instead

**Review by natetarrh — CHANGES_REQUESTED** [review]
thanks for the fixes!

File: ADXL362.py (hunk 1 above)
Comment (Human) [reply ADXL362.py:173]: Cool. I'll submit new PRs, since I just bundled a few of them together in the same branch.

**Comment by Human** [comment]
Closing this PR to spawn 3 new ones.

**Pull request closed without merging**